### PR TITLE
fix: do not default to `noverify` for `bindings` sessions in CLI.

### PR DESCRIPTION
**Pull request by ioga**
## Description

OSS user has reported we were defaulting no `noverify` in `setup_session` helper: https://determined-community.slack.com/archives/CV3MTNZ6U/p1650642072594889
We don't do it in cli cert setups elsewhere, and all of `noverify`, `cert_path`, and `cert_name` options on `args` seem to be dead. 

## Test Plan

CLI shouldn't print urllib's noverify warnings when used with HTTPS cluster.

## Checklist

- [ ] User-facing API changes need the "User-facing API Change" label.
- [ ] Release notes should be added as a separate file under `docs/release-notes/`.
See [Release Note](../docs/release-notes/README.md) for details.
- [ ] Licenses should be included for new code which was copied and/or modified from any external code.

### CODE REVIEW

File: docs/release-notes/fix-noverify.txt [hunk 1]
```diff
@@ -0,0 +1,5 @@
+:orphan:
+
+**Bug Fixes**
+
+-  CLI: Stop using SSL "noverify" option for some API requests.
```
Comment (gh-determined-ai) [inline docs/release-notes/fix-noverify.txt:5]: This needs to be more informative. Can you say, even generally, which API requests? Why was this change needed?

**Review by gh-determined-ai — CHANGES_REQUESTED** [review]

File: docs/release-notes/fix-noverify.txt (hunk 1 above)
Comment (ioga) [reply docs/release-notes/fix-noverify.txt]: updated.

**Review by gh-determined-ai — APPROVED** [review]

**Review by gh-determined-ai — APPROVED** [review]

**Review by hamidzr — APPROVED** [review]
